src/lib/hooks/useFilteredFields.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+import { EProjectFields } from '../consts/projects';
+
+export const useFilteredFields = (
+  watchedFields: any,
+  setActiveFilters: any,
+) => {
+  useEffect(() => {
+    const activeFilterFields: Partial<Record<EProjectFields, boolean>> = {};
```

Comment:
instead of EProjectFields add generic type to the hook

Reply:
see comment https://github.com/shlomogcap/b-cont-react/pull/14#discussion_r1252624998